views/index.ejs

```diff
@@ -7,31 +8,59 @@
-                <div class="card-section" data-section="<%= section.name %>">
-                    <% if (section.name.toLowerCase() === 'official') { %>
-                        <div class="screeps-logo">
-                            <img src="<%= section.logo %>" alt="Screeps Logo" />
-                        </div>
-                    <% } else { %>
-                        <h2><%= section.name %></h2>
+                <% if(serverList.length===0){ %>
+                    <h1> Could not load Server List</h1>
+                    <% } %>
+                        <% serverList.forEach(section=> { %>
+                            <div class="card-section" data-section="<%= section.name %>">
+                                <% if (section.name.toLowerCase()==='official' ) { %>
+                                    <div class="screeps-logo">
+                                        <img src="<%= section.logo %>" alt="Screeps Logo" />
+                                    </div>
+                                    <% } else { %>
+                                        <h2>
+                                            <%= section.name %>
+                                        </h2>
+                                        <% } %>
+                                            <% section.servers.forEach(server=> { %>
+                                                <div class="server">
+                                                    <a href="<%= server.url %>">
+                                                        <button>
+                                                            <%= server.name %>
+                                                        </button>
+                                                    </a>
+                                                    <span class="status-circle" data-api="<%= server.api %>"></span>
+                                                </div>
+                                                <% }) %>
+                            </div>
+                            <% }) %>
+        </div>
+        <div class="card community-card">
+            <% if(communityPages.length===0){ %>
+                <h1> Could not load Community Content</h1>
+                <% } else{ %>
+                    <h2>Community Content</h2>
                     <% } %>
-                    <% section.servers.forEach(server => { %>
-                        <div class="server">
-                            <a href="<%= server.url %>">
-                                <button><%= server.name %></button>
-                            </a>
-                            <span class="status-circle" data-api="<%= server.api %>"></span>
+                        <div class="comm-container">
+                            <% communityPages.forEach(page=> { %>
+                                <div class="comm-page">
+                                    <a href="<%= page.url %> " target="_blank" rel="noopener noreferrer">
```

Comment:
There's a space character at the end of this `href` value. Would you mind removing that?

Reply:
@Gadjung

Reply:
should be fixed